src/programs/pkexec.c

```diff
@@ -664,6 +664,28 @@ main (int argc, char *argv[])
         argv[n] = path;
       }
     }
+  if (1)
```

Comment:
Please explain.

Reply:
Heh. Pure indentation thing.

I didn't realise I left it in there like that. Can remove that and de-indent. Especially since I later realised I didn't need to declare `s`.